src/components/Navbar/SideNav.css

```
@@ -0,0 +1,9 @@
+.SideNav {
```

I suggest following the BEM naming convention for `css`
one of the reasons we put `-` between words in the classes is to distinguish between css classes and other JS variables
```suggestion
.side-nav {
```